kotlin/workflow-ui/core-android/src/main/java/com/squareup/workflow/ui/LayoutRunner.kt

```diff
@@ -51,6 +51,27 @@ import kotlin.reflect.KClass
  *    val TicTacToeViewBuilders = ViewRegistry(
  *        NewGameLayoutRunner, GamePlayLayoutRunner, GameOverLayoutRunner
  *    )
+ *
+ * ## AndroidX ViewBinding
+ *
+ * [LayoutRunner]s also support AndroidX ViewBindings. There is a [LayoutRunner.bind] overload that
+ * takes a reference to an [androidx.viewbinding.ViewBinding]'s `inflate` method and a
+ * [LayoutRunner] constructor that accepts an [androidx.viewbinding.ViewBinding] instead of a
+ * [View].
+ *
+ *   class HelloLayoutRunner(private val binding: HelloGoodbyeLayoutBinding) : LayoutRunner<Rendering> {
+ *
+ *     override fun showRendering(rendering: Rendering) {
+ *       binding.messageView.text = rendering.message
+ *       binding.messageView.setOnClickListener { rendering.onClick(Unit) }
+ *     }
+ *
+ *     companion object : ViewBinding<Rendering> by bind(
+ *         HelloGoodbyeLayoutBinding, ::HelloLayoutRunner
+ *     )
+ *   }
+ *
+ * If the view does not need to be initialized, the [bindViewBinding] function can be used instead.
```

Comment:
And we _really_ have to rename our own `ViewBinding` class. I don't think we should cram that into this PR, but we should do it before the next release. :(

Reply:
I didn't want to hang this off `LayoutRunner` because its API doesn't have anything to do with `LayoutRunner`, it's just an implementation detail. But "`LayoutRunner` as one-stop-shopping for dealing with Layout files" makes sense, so I'll change it.